index.js

```diff
@@ -6,6 +6,9 @@
 // // => 2
 // console.log("%c----------", "color: red") 
 
+counter = 1
+counter = 2
```

Comment:
use `let` here to declare the variable. `let` is a keyword that we can use to tell JavaScript explicitly that we are creating a variable. `let` will give the variable "block" scope meaning that the variable will only exist within the two curly braces in which it's defined. `let` allows us to re-assign the variable as well.

```suggestion
let counter = 1
counter = 2
```